tests/compatibility_tests/torch_comp_tests/compatibility_load_model_test.py

```diff
@@ -0,0 +1,238 @@
+# Copyright 2023 Sony Semiconductor Israel, Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+import torch
+
+from mct_quantizers.pytorch.quantizers import WeightsPOTInferableQuantizer, WeightsSymmetricInferableQuantizer, \
+    WeightsUniformInferableQuantizer, WeightsLUTPOTInferableQuantizer, WeightsLUTSymmetricInferableQuantizer, \
+    ActivationPOTInferableQuantizer, ActivationUniformInferableQuantizer, \
+    ActivationSymmetricInferableQuantizer
+from tests.compatibility_tests.torch_comp_tests.base_activation_compatibility_test import \
+    BaseActivationQuantizerLoadAndCompareTest
+from tests.compatibility_tests.torch_comp_tests.base_weights_compatibility_test import BaseWeightsQuantizerLoadAndCompareTest
+from tests.compatibility_tests.torch_comp_tests.onnx_utils import _get_qparams_from_input_tensors_for_single_quantizer, \
+    _get_qparams_from_attributes_for_single_quantizer
+
+
+class WeightsPOTQuantizerLoadAndCompareTest(BaseWeightsQuantizerLoadAndCompareTest):
+
+
+    def setUp(self):
+        self.quantizer_type = WeightsPOTInferableQuantizer
+
+    def test_weights_uniform_quantizer(self):
+        self.conv_test(self.quantizer_type)
+        self.convtrans_test(self.quantizer_type)
+        self.dense_test(self.quantizer_type)
+
+    def _check_quantizer_init_from_onnx_model(self, onnx_file_path):
```

Comment:
this method is duplicated across several classes, maybe put it in the parent class (one for lut and one for regular quantizer test?) or extract its content to an external function outside the class and call it from all places (I think this method isn't class-dependent, i.e., can be static or external, isn't it?)